partiql-plan/src/main/java/org/partiql/plan/Plan.java

```diff
@@ -0,0 +1,25 @@
+package org.partiql.plan;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * A plan holds operations that can be executed.
+ */
+public interface Plan {
+
+    /**
+     * @return version for serialization and debugging.
+     */
+    @NotNull
+    default public Version getVersion() {
+        return Version.UNKNOWN();
+    }
+
+    /**
+     * @return statement actions to execute.
+     */
+    @NotNull
+    public List<Action> getActions();
```

Comment:
The list here. Is this supposed to represent essentially sub-plans?

Reply:
Initially for a transaction, but I believe now that "transaction" should be an "action" which holds more actions. I will update this